mypy.ini

```diff
@@ -0,0 +1,2 @@
+[mypy]
```

Comment:
Why do we need this?
I'm not in favor of hiding missing imports, which is a problem we must spot.

Reply:
I copied this from `taipy-gui/mypy.ini`.  I needed to do this because in order for the pre-commit to work, it runs `mypy`, and I had to copy over the contents of the other projects into the `taipy` project as follows:

-  `taipy-core/src/taipy/core` -> `taipy/src/taipy/core`
-  `taipy-gui/src/taipy/gui` -> `taipy/src/taipy/gui`
-  `taipy-rest/src/taipy/rest` -> `taipy/src/taipy/rest`

Without `mypy.ini` in there, then it complains about missing imports from other packages.